src/components/editor/Editor.jsx

```diff
@@ -0,0 +1,55 @@
+import ReactQuill from "react-quill";
+import "react-quill/dist/quill.snow.css";
+import "./editor.css";
+
+export function Editor() {
+  const formats = [
+    "header",
+    "font",
+    "size",
+    "bold",
+    "italic",
+    "underline",
+    "strike",
+    "blockquote",
+    "list",
+    "bullet",
+    "indent",
+    "link",
+    "image",
+    "video",
+  ];
+
+  const modules = {
+    toolbar: [
+      [{ header: "1" }, { header: "2" }, { font: [] }],
+      [{ size: [] }],
+      ["bold", "italic", "underline", "strike", "blockquote"],
+      [
+        { list: "ordered" },
+        { list: "bullet" },
+        { indent: "-1" },
+        { indent: "+1" },
+      ],
+      ["link", "image", "video"],
+      ["clean"],
+    ],
+    clipboard: {
+      // toggle to add extra line breaks when pasting HTML:
+      matchVisual: false,
+    },
+  };
+
+  return (
+    <>
+      <ReactQuill
+        theme="snow"
+        placeholder="Add a note"
+        //   value={value}
+        //   onChange={setValue}
```

Comment:
Can remove these commented values.

Reply:
These changes will come in upcoming branch so leaving it as is.